src/docs/concepts-development/sitemap/index.md

```diff
@@ -0,0 +1,55 @@
+---
+title: Sitemap support in Sharetribe
+slug: sitemap-in-sharetribe
+updated: 2023-10-12
+category: concepts-development
+ingress:
+  This guide describes how the Sharetribe tooling supports creating a
+  sitemap for your marketplace.
+published: true
+---
+
+Having a sitemap on your marketplace helps search engines to process
+your website correctly. For a traditional website, sitemaps are
+relatively easy to make, because all of your routes are built in to the
+application.
+
+However, on a marketplace, it is likely that you will have listing URLs
+that include dynamic data from each listing – and you still want search
+engines to index those pages.
+
+<info>
+
+The Sharetribe Web Template has tooling to create a dynamic sitemap by
+default. Read more about the template sitemap structure and logic:
+**[Sitemap in Sharetribe Web Template](/sitemap-in-template)**
+
+</info>
+
+If you are creating your own custom application, you can consider
+whether you want to use the same practices that the web template uses to
+craft the sitemap.
+
+## Separate sitemaps for static and dynamic content
+
+The web template exposes three different sub-sitemaps from
```

Comment:
Comment:
How this is worded makes it seem that this information is web-template specific. Is the idea here to use the approach the web-template uses as an example for non-template developers? What do you think about using a more general approach, e.g.:

Static content sitemaps help search engines quickly index unchanging pages like your "About" page, while dynamic content sitemaps focus on frequently updated or user-generated content, such as listings in your marketplace. For example, the Sharetribe Web Template generates three different sub-sitemaps linked from _sitemap-index.xml_, all with slightly different data: